invenio_theme/assets/semantic-ui/js/invenio_theme/truncate.js

```diff
@@ -0,0 +1,24 @@
+import $ from "jquery";
+
+$(".truncate-button").on("click", function () {
+  const $clickButton = $(this);
+  const siblings = $clickButton.siblings();
+  siblings.first().toggleClass("truncate-2");
+  $clickButton.toggle();
```

Comment:
$(this).toggle();

Reply:
Why this one? it looks ok to me.

Reply:
it is connected with the previous suggestion, that the separate variable (`clickButton`) is redundant since we have `this`